flink-python/pyflink/table/module.py

```diff
@@ -34,6 +37,7 @@ def __init__(self, j_module):
         self._j_module = j_module
 
 
+@PublicEvolving()
 class HiveModule(Module):
```

Comment:
Not sure why this is here. The Hive module got factored out of the Flink core repository? Maybe a followup issue to remove it? Maybe don't annotate it?